<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_select">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>911</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="intro_tab">
     <attribute name="title">
      <string>Intro</string>
     </attribute>
     <widget class="QLabel" name="intro_text_1">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>22</y>
        <width>871</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/This Is OMEGA 2.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="scenario_tab">
     <attribute name="title">
      <string>Scenario</string>
     </attribute>
     <widget class="QPlainTextEdit" name="scenario_file_1_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>771</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="scenario_file_1_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Scenario File</string>
      </property>
     </widget>
     <widget class="QLabel" name="working_directory_1_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Working Directory</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="working_directory_1_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>771</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="input_files_tab">
     <attribute name="title">
      <string>Input Files</string>
     </attribute>
     <attribute name="toolTip">
      <string>Vehicle Type</string>
     </attribute>
     <widget class="QTableWidget" name="input_file_table">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>891</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="output_files_tab">
     <attribute name="title">
      <string>Output Files</string>
     </attribute>
     <widget class="QTableWidget" name="output_file_table">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>891</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="electrification_tab">
     <attribute name="title">
      <string>Electrification</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="road_load_tab">
     <attribute name="title">
      <string>Road Load</string>
     </attribute>
     <widget class="QLabel" name="mass_reduction_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass Reduction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="mass_reduction_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="mass_reduction_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="mass_reduction_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mass_reduction_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="mass_reduction_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mass_reduction_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="mass_reduction_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="rolling_reduction_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="rolling_reduction_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="rolling_reduction_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rolling_reduction_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="rolling_reduction_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="rolling_reduction_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="rolling_reduction_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="aero_reduction_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="aero_reduction_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="aero_reduction_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="aero_reduction_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="aero_reduction_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aero_reduction_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="aero_reduction_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>114</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="engine_sizing_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="engine_sizing_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="engine_sizing_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="engine_sizing_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="engine_sizing_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="engine_sizing_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="engine_sizing_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="rolling_reduction_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rolling Reduction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="aero_reduction_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Aero Reduction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="engine_sizing_label">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Engine Sizing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_mass_reduction_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>212</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Mass Reduction</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_of_iterations">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_rolling_reduction_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Rolling Reduction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_engine_sizing_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Engine Sizing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_aero_reduction_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Aero Reduction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calculate_iterations_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate # of Iterations</string>
      </property>
     </widget>
     <widget class="QLabel" name="number_of_iterations_label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string># of Iterations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="calculation_time_label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculation Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="calculation_time">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EPA OMEGA Model</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="epa_logo_label1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>EPA Logo</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/green_car.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu1">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_file"/>
    <addaction name="action_open_file"/>
    <addaction name="separator"/>
    <addaction name="action_save_file"/>
    <addaction name="action_save_file_as"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionAbout_OMEGA_2"/>
   </widget>
   <addaction name="menuMenu1"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionEngines">
   <property name="text">
    <string>Engines</string>
   </property>
  </action>
  <action name="actionTransmissions">
   <property name="text">
    <string>Transmissions</string>
   </property>
  </action>
  <action name="actionRoad_Loads">
   <property name="text">
    <string>Road Loads</string>
   </property>
  </action>
  <action name="action_open_file">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_save_file">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_save_file_as">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="action_new_file">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionAbout_OMEGA_2">
   <property name="text">
    <string>About OMEGA</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
